### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by AndriiZakharenko**
- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,103 @@
...
+  margin-left: 20px;
+}
+
+.header__link {
+  font-weight: 500;
+  font-size: var(--main);
+  line-height: 60px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
```
Comment (polosanya) [inline src/style.css:73]: Pay attention to the CSS properties order
https://9elements.com/css-rule-order/

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,103 @@
...
+  height: 60px;
+  transition: color 0.3s ease;
+}
+
+.header__link:hover {
+  color: var(--blue);
+}
+
+.header__link:active {
+  color: var(--blue);
```
Comment (polosanya) [inline src/style.css:85]: You can combine the selectors with same properties

**Review by polosanya — CHANGES_REQUESTED** [review]
Pay attention that the header should have box-shadow property
image
Also, please don't forget to add demo and test links to the PR description

File: src/style.css (hunk 2 above)
Comment (AndriiZakharenko) [reply src/style.css:85]: done

File: src/style.css (hunk 1 above)
Comment (AndriiZakharenko) [reply src/style.css:73]: done

**Comment by AndriiZakharenko** [comment]
- [DEMO LINK](https://AndriiZakharenko.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://AndriiZakharenko.github.io/layout_moyo-header/report/html_report/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

**Pull request closed without merging**